.github/workflows/main.yml

```diff
@@ -19,7 +19,7 @@ on:
       - submitted
 
 env:
-  XCLIM_TESTDATA_BRANCH: v2023.12.14
+  XCLIM_TESTDATA_BRANCH: cleanup
```

Comment:
what does this mean? XCLIM_TESTDATA_BRANCH will still exist no? The functions to interact with it have simply changed. Should we not keep some versioning like before?